app/models/vehicles.py

```diff
@@ -4,73 +4,146 @@
 from app.models.db import vehicles
 
 
-class ErrVehicleNotFound(Exception):
+class ErrNotFound(Exception):
+    pass
+
+
+class ErrInvalidParameters(Exception):
     pass
 
 
 class ErrVehicleAlreadyExisit(Exception):
     pass
 
 
-# CRUD functions
 def insert(
-    shop_id=None,
+    shop_id="",
     make="",
     model="",
     year="",
-    price="",
+    trim="",
+    engine="",
     title="",
     condition="",
+    drive="",
+    fuel="",
+    transmission="",
+    exterior="",
+    interior="",
```

Comment:
thanks for adding those extra fields, appreciate it. 
We still need to have the `odometer`

Reply:
oh we also need the price :D

Reply:
oh here @qangdev I left the comment here for the fields we need to add

Reply:
`price` and `odometer` have been added.